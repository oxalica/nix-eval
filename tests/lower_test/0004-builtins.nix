[
  # Just like keywords
  true false
  # https://nixos.org/manual/nix/stable/#ssec-builtins
  abort
  baseNameOf
  builtins
  derivation
  dirOf
  fetchTarball
  import
  isNull
  map
  removeAttrs
  throw
  toString
]
