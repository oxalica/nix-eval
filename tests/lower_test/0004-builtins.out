Ok(
    Expr(
        List {
            items: [
                Expr(
                    Literal(
                        Bool(
                            true,
                        ),
                    ),
                ),
                Expr(
                    Literal(
                        Bool(
                            false,
                        ),
                    ),
                ),
                Expr(
                    Builtin(
                        Abort,
                    ),
                ),
                Expr(
                    Builtin(
                        BaseNameOf,
                    ),
                ),
                Expr(
                    Builtin(
                        Builtins,
                    ),
                ),
                Expr(
                    Builtin(
                        Derivation,
                    ),
                ),
                Expr(
                    Builtin(
                        DirOf,
                    ),
                ),
                Expr(
                    Builtin(
                        FetchTarball,
                    ),
                ),
                Expr(
                    Builtin(
                        Import,
                    ),
                ),
                Expr(
                    Builtin(
                        IsNull,
                    ),
                ),
                Expr(
                    Builtin(
                        Map,
                    ),
                ),
                Expr(
                    Builtin(
                        RemoveAttrs,
                    ),
                ),
                Expr(
                    Builtin(
                        Throw,
                    ),
                ),
                Expr(
                    Builtin(
                        ToString,
                    ),
                ),
            ],
        },
    ),
)